packager/react-packager/src/Server/__tests__/Server-test.js

```diff
@@ -207,57 +236,57 @@ describe('processRequest', () => {
   describe('/onchange endpoint', () => {
     var EventEmitter;
     var req;
-    var res;
+    var body;
```

Comment:
can you change this back to `res`? the model is - there's a response object that has several methods on it like `end` and one field of it (for testing purposes) is a string called `body` which contains the body of the HTTP response